aioesphomeapi/_frame_helper/base.py

```diff
@@ -19,8 +19,10 @@
 
 WRITE_EXCEPTIONS = (RuntimeError, ConnectionResetError, OSError)
 
+_int = int
 
-class APIFrameHelper(asyncio.Protocol):
+
+class APIFrameHelper:
```

Comment:
`asyncio.Protocol` is optional but since cython only allows one base class, we can removed it.